coordinatorlib/src/main/scala/com/socrata/datacoordinator/secondary/SecondaryInfo.scala

```diff
@@ -1,11 +1,31 @@
 package com.socrata.datacoordinator.secondary
 
+import java.sql.ResultSet
+
 import org.joda.time.DateTime
 
-case class SecondaryConfigInfo(storeId: String, nextRunTime: DateTime, runIntervalSeconds: Int)
+case class SecondaryInstanceInfo(storeId: String, nextRunTime: DateTime, runIntervalSeconds: Int)
+object SecondaryInstanceInfo {
+  def apply(rs: ResultSet): SecondaryInstanceInfo =
+    SecondaryInstanceInfo(rs.getString("store_id"),
```

Comment:
Should this sql specific stuff be somewhere in the sql package instead of the base object?